run_unittests.py

```diff
@@ -5076,6 +5076,14 @@ def test_pkg_config_option(self):
         testdir = os.path.join(self.unit_test_dir, '55 pkg_config_path option')
         self.init(testdir, extra_args=['-Dpkg_config_path=' + os.path.join(testdir, 'extra_path')])
 
+    def test_std_remains(self):
+        # C_std defined in project options must be in effect also when cross compiling.
```

Comment:
This should not say "also when cross compiling". This one tests native compilation, the other one is cross compilation

Reply:
Thanks. Should be fixed.